lib/saharspec.rb

```diff
@@ -1,8 +1,6 @@
 # frozen_string_literal: true
 
-defined?(RSpec) or
-  fail 'RSpec is not present in the current environment, check that `rspec` ' \
-        'is present in your Gemfile and is in the same group as `saharspec`' \
+require 'rspec/core'
```

Comment:
If no "rspec/core" defined it will raise a clear message about that:

```
$ RAILS_ENV=test rails db:migrate
rails aborted!
LoadError: cannot load such file -- rspec/core
```